galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/GenericMethodWrapper.java

```diff
@@ -66,6 +64,7 @@ public void invoke(@NotNull TestRunManagers managers, Object testClassObject, Ge
             Result ignored = managers.anyReasonTestMethodShouldBeIgnored(new GalasaMethod(this.excecutionMethod, null));
             if (ignored != null) {
                 this.result = ignored;
```

Comment:
why do we need to set this result globally?

Reply:
We need this as after we return out of invoke in GenericMethodWrapper we go back into TestClassWrapper, and on line 213 it calls a method which needs the global result variable. Without setting result we get a null pointer.